# The state of the 6D base of a system

geometry_msgs/Pose      pose        # The 6D pose of the base
geometry_msgs/Twist    twist        # The 6D velocity of the base in Base Frame
geometry_msgs/Accel    accel        # The 6D acceleration of the base in Base Frame